src/store/utils.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  WorkerResponseParams,
+  CommandWorkerRequestParams,
+  WorkerRequestParams,
+  Commands,
+} from "@/packages/web-worker/types";
+
+const RESPONSE_WAIT_TIMEOUT = 5000;
```

Comment:
Response timeout, although generally is a good idea, I'd put a much larger number in here (like `60_000`). 

Right now the issue is that e.g. loading takes time as it has fetching wasm file inside so depending on server + network conditions can result in much longer wait time. 

Other remark is that after timeout we should get the UI back to the state it was initially. In the case above it stucks on spinning circles.

Reply:
I increased timeout to 60s. We can have a longer timeout for commands like load. Others may be shorter. 60s for each command may confuse user and we can predict how much time should take specific commands. I set timeout as a promise reject to handle it similar to `commandStatus: 'error"` and now when I think about it maybe it should be `commandStatus: 'error"` with a timeout error. In such case handling would be similar to standard error from worker.